frontend/src/pages/Clusters/ClusterActions.js

```diff
@@ -64,8 +64,11 @@ export default function ClusterActions () {
       loadTemplate(jsyaml.load(configuration));
     });
   }
+
   const deleteCluster = () => {
-    setState(['app', 'clusters', 'clusterDelete', 'dialog'], true)
+    console.log(`Deleting: ${clusterName}`);
+    DeleteCluster(clusterName, (resp) => {DescribeCluster(clusterName); ListClusters()});
+    hideDialog('deleteCluster');
```

Comment:
I think this call to `hideDialog` should probably go into the callback above (?)

Reply:
Hmm... I think this should work as you provided. You can put a log in the anonymous function, but presumably the other functions are being called (e.g. `DescribeCluster` and `ListClusters`)  and `Delete success` is being printed?